movie-app1.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const selector = document.querySelector('#select-movie')
+    const form = document.querySelector('#form')
+
+    selector.addEventListener('change', () => {
+       if (mainDiv === '') {
+           displayMovieContent(selector.selectedIndex - 1)
+       } else {
+           displayMovieContent(selector.selectedIndex - 1)
+       }
+    }) 
+    form.addEventListener('submit', postReview)
+})
+
+const mainDiv = document.querySelector('#maincontent')
```

Comment:
When I console log "mainDiv" here it would not run since it is trying to grab elements before all DOM elements are loaded. It is better practice to include all codes inside DOMContentLoaded since that will ensure all DOM elements to be loaded and then be used as global variables.